lambdas/data-migration2/src/errors.ts

```diff
@@ -0,0 +1,3 @@
+import { createErrorType } from '@cumulus/errors';
+
+export const RecordAlreadyMigrated = createErrorType('RecordAlreadyMigrated');
```

Comment:
I should have just done this to begin with, but since we're using this same error type in data-migration1 and data-migration2, let's move this to being exported from the `errors` package and just use it in both lambdas

Reply:
Yup, makes sense! I moved the `RecordAlreadyMigrated` to the Errors package, removed errors.ts from both data-migration1 and data-migration2 Lambdas and updated the tests.